config/index.ts

```diff
@@ -20,11 +23,68 @@ const config: NetworkConfig = {
     process.env.CHAINLINK_ROPSTEN_PRICE_FEED || '0x0000000000000000000000000000000000000000',
+
+  CHAINLINK_MAINNET_TOKEN:
+    process.env.CHAINLINK_MAINNET_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_KOVAN_TOKEN: process.env.CHAINLINK_KOVAN_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_GOERLI_TOKEN: process.env.CHAINLINK_GOERLI_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_RINKEBY_TOKEN:
+    process.env.CHAINLINK_RINKEBY_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_ROPSTEN_TOKEN:
+    process.env.CHAINLINK_ROPSTEN_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_MATIC_TOKEN:
+    process.env.CHAINLINK_MATIC_TOKEN || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_MATIC_MUMBAI_TOKEN:
+    process.env.CHAINLINK_MATIC_MUMBAI_TOKEN || '0x0000000000000000000000000000000000000000',
+
+  CHAINLINK_MAINNET_VRF_COORD:
+    process.env.CHAINLINK_MAINNET_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_KOVAN_VRF_COORD: process.env.CHAINLINK_KOVAN_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_GOERLI_VRF_COORD: process.env.CHAINLINK_GOERLI_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_RINKEBY_VRF_COORD:
+    process.env.CHAINLINK_RINKEBY_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_ROPSTEN_VRF_COORD:
+    process.env.CHAINLINK_ROPSTEN_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_MATIC_VRF_COORD:
+    process.env.CHAINLINK_MATIC_VRF_COORD || '0x0000000000000000000000000000000000000000',
+  CHAINLINK_MATIC_MUMBAI_VRF_COORD:
+    process.env.CHAINLINK_MATIC_MUMBAI_VRF_COORD || '0x0000000000000000000000000000000000000000',
+
+  CHAINLINK_MAINNET_VRF_KEY_HASH:
+    process.env.CHAINLINK_MAINNET_VRF_KEY_HASH || '0x00',
+  CHAINLINK_KOVAN_VRF_KEY_HASH: process.env.CHAINLINK_KOVAN_VRF_KEY_HASH || '0x00',
+  CHAINLINK_GOERLI_VRF_KEY_HASH: process.env.CHAINLINK_GOERLI_VRF_KEY_HASH || '0x00',
+  CHAINLINK_RINKEBY_VRF_KEY_HASH:
+    process.env.CHAINLINK_RINKEBY_VRF_KEY_HASH || '0x00',
+  CHAINLINK_ROPSTEN_VRF_KEY_HASH:
+    process.env.CHAINLINK_ROPSTEN_VRF_KEY_HASH || '0x00',
+  CHAINLINK_MATIC_VRF_KEY_HASH:
+    process.env.CHAINLINK_MATIC_VRF_KEY_HASH || '0x00',
+  CHAINLINK_MATIC_MUMBAI_VRF_KEY_HASH:
+    process.env.CHAINLINK_MATIC_MUMBAI_VRF_KEY_HASH || '0x00',
+
+  CHAINLINK_MAINNET_VRF_FEE:
+    process.env.CHAINLINK_MAINNET_VRF_FEE || '0',
+  CHAINLINK_KOVAN_VRF_FEE: process.env.CHAINLINK_KOVAN_VRF_FEE || '0',
+  CHAINLINK_GOERLI_VRF_FEE: process.env.CHAINLINK_GOERLI_VRF_FEE || '0',
+  CHAINLINK_RINKEBY_VRF_FEE:
+    process.env.CHAINLINK_RINKEBY_VRF_FEE || '0',
+  CHAINLINK_ROPSTEN_VRF_FEE:
+    process.env.CHAINLINK_ROPSTEN_VRF_FEE || '0',
+  CHAINLINK_MATIC_VRF_FEE:
+    process.env.CHAINLINK_MATIC_VRF_FEE || '0',
+  CHAINLINK_MATIC_MUMBAI_VRF_FEE:
+    process.env.CHAINLINK_MATIC_MUMBAI_VRF_FEE || '0',
```

Comment:
so env vars or 0? why not null?

Reply:
yup, i try to fail open when i can with these vars but sometimes you can't